subprojects/build-scan-performance/src/testFixtures/groovy/org/gradle/performance/AbstractBuildScanPluginPerformanceTest.groovy

```diff
@@ -28,16 +28,12 @@ import org.gradle.performance.measure.MeasuredOperation
 import org.gradle.performance.results.BaselineVersion
 import org.gradle.performance.results.BuildScanResultsStore
 import org.gradle.performance.results.CrossBuildPerformanceResults
-import org.gradle.performance.results.GradleProfilerReporter
-import org.gradle.test.fixtures.file.CleanupTestDirectory
 import org.gradle.test.fixtures.file.TestNameTestDirectoryProvider
 import org.junit.Rule
 import spock.lang.AutoCleanup
 import spock.lang.Shared
-import spock.lang.Specification
 
-@CleanupTestDirectory
```

Comment:
Why this is removed?

Reply:
Seems like we're not using `temporaryFolder.testDirectory` any more, does gradle-profiler clean it? What if we keep this annotation, I assume it does nothing in this case?

Reply:
I added the annotation to the superclass. That should work, right? TemporaryFolder.testDirectory is always removed, the annotation causes to not remove it if the test fails, right?